### PR TITLE
Add '.exe' on windows in make-targets

### PR DESCRIPTION
**Pull request by jstroem**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -25,11 +25,18 @@ ESP32_WIFI_SSID=
 ESP32_WIFI_PASSWORD=
 ESP32_PORT=
 
+
+ifeq ($(OS),Windows_NT)
```
Comment (kasperl) [inline Makefile:29]: Is this always the right test?

File: Makefile [hunk 2]
```diff
@@ -25,11 +25,18 @@ ESP32_WIFI_SSID=
 ESP32_WIFI_PASSWORD=
 ESP32_PORT=
 
+
```
Comment (kasperl) [inline Makefile:28]: Drop one newline.

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: Makefile (hunk 1 above)
Comment (jstroem) [reply Makefile:29]: The internet says so.

**Pull request merged**